tests/haskell/GhcModBearTest.py

```diff
@@ -0,0 +1,51 @@
+from bears.haskell.GhcModBear import GhcModBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+good_file1 = """
+main :: IO()
+main = putStrLn "Hello World!"
+"""
+
+good_file2 = """
+main :: IO()
+main = interact lineC
+    where lineC input = show (length(lines input))
+"""
+
+good_file3 = """
+lastButOne :: [a] -> a
+lastButOne xs | (length xs >= 2) = xs !! ((length xs) - 2)
+              | otherwise = error "Not possible"
+
+main :: IO()
+main = do
+    print(lastButOne(["asd", "ASd"]))
+"""
+
+bad_file1 = """
+main :: IO()
+main = putStrLn 3
+"""
+
+bad_file2 = """
+main :: IO()
+main = interact lineC
+    where lineC input = show (length(lines input)
+"""
+
+bad_file3 = """
+lastButOne :: a -> a
+lastButOne xs | (length xs >= 2) = xs !! ((length xs) - 2)
+              | otherwise = error "Not possible"
+
+main :: IO()
+main = do
+    print(lastButOne(["asd", "ASd"]))
+"""
+
```

Comment:
hmm let's use a test class with a skipDecorator so we know whether it's skipping because of a depend or some internal logic error. See http://api.coala.io/en/latest/Developers/Testing_Bears.html